apis/Google.Cloud.Compute.V1/Google.Cloud.Compute.V1.IntegrationTests/AcceptanceTest.cs

```diff
@@ -122,4 +122,248 @@ private Operation PollForCompletion(Operation operation, string alias)
             return operation;
         }
     }
+    [Collection(nameof(ComputeFixture))]
+    public sealed class InstancesTests{
+        private readonly ITestOutputHelper _output;
+        private readonly ComputeFixture _fixture;
+
+        public InstancesTests(ComputeFixture fixture, ITestOutputHelper output) =>
+            (_fixture, _output) = (fixture, output);
+
+        [Fact]
+        public void CreateInstance(){
+            string projectId = _fixture.ProjectId;
+            string zone = _fixture.Zone;
+            string diskImage = "projects/debian-cloud/global/images/family/debian-10";
+            string machineType = $"https://www.googleapis.com/compute/v1/projects/{projectId}/zones/{zone}/machineTypes/n1-standard-1";
+            InstancesClient instancesClient = InstancesClient.Create();
+            var instanceName = _fixture.GenerateResourceId();
+
+            CreateInstanceNonAsciiDescription();
+            FetchInstance();
+            PatchInstance();
+            DeleteInstance();
+
+            void CreateInstanceNonAsciiDescription(){
+                AttachedDiskInitializeParams init_params = new AttachedDiskInitializeParams
```

Comment:
Nit: init_params => initParams - or better yet, just inline this into AttachedDisk object initializer.